### DISCUSSION TITLE
Project Insights seems not to filter by iteration

### DISCUSSION THREAD

**Discussion by cachedout** [community/community#43186] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

Hello,

I've discovered some odd behavior with GitHub Projects and I would like some feedback from folks about whether this is expected or whether this is a potential bug.

Given a project that has more than one iteration, and creating a chart with the following configuration:

<img width="402" alt="Screen Shot 2023-01-03 at 10 17 03" src="https://user-images.githubusercontent.com/111616/210328870-6000eea5-f0f5-4787-a331-d42d2ec54959.png">

And with the following filter set:

<img width="258" alt="Screen Shot 2023-01-03 at 10 17 48" src="https://user-images.githubusercontent.com/111616/210329000-30fb3ee5-24e5-4ddf-a904-6941bbbf1b2b.png">

My expectation is that the resulting graph would contain _only_ issues in the current iteration. However, that is not the case. It seems, instead, that the `Iteration' filter has no effect whatsoever:

<img width="1021" alt="Screen Shot 2023-01-03 at 10 19 03" src="https://user-images.githubusercontent.com/111616/210329191-9f76e877-10a7-4386-afa1-33ff5edf4470.png">

In the above graph, I believe I should see only the seven issues in the current iteration but as you can see, it finds over 200 issues.

<img width="159" alt="Screen Shot 2023-01-03 at 10 19 33" src="https://user-images.githubusercontent.com/111616/210329263-0a05fe8d-87ee-450c-90ae-a16fcfe05a1e.png">

Can anybody comment on this behavior? Bug or user error? Thanks.

**Comment by kallehauge** [comment, marked as answer]
It sounds very related to this one: https://github.com/orgs/community/discussions/42422

We're also experiencing it right now. All filtering in insights charts seems to be bugged right now - but luckily they still work in views. Just annoying when we have to do end of year reporting 😅

**Reply by cachedout** [reply]
Thanks, @kallehauge . Looks like the same problem.

**Reply by mtcA6** [reply]
@kallehauge What do you mean 'sitll work in views'

**Reply by kallehauge** [reply]
@mtcA6 it has been a while but I'm pretty sure this was a temporary issue where all filtering didn't work when creating insights charts, but it was still possible to use filters on project views (read: table, board, and roadmap layout on the project itself)

**Reply by mtcA6** [reply]
Might not be so temporary, I'm pretty sure I'm having this issue in my project rn.
